ggml/src/ggml-cpu/ggml-cpu.cpp

```diff
@@ -658,10 +552,15 @@ static ggml_backend_feature * ggml_backend_cpu_get_features(ggml_backend_reg_t r
 
 static void * ggml_backend_cpu_get_proc_address(ggml_backend_reg_t reg, const char * name) {
     if (strcmp(name, "ggml_backend_set_n_threads") == 0) {
-        return (void *)ggml_backend_cpu_set_n_threads;
+        ggml_backend_set_n_threads_t fct = ggml_backend_cpu_set_n_threads;
+        return (void *)fct;
     }
     if (strcmp(name, "ggml_backend_dev_get_extra_bufts") == 0) {
-        return (void *)ggml_backend_cpu_get_extra_bufts;
+        // why not add it to the global API?
+        //static const struct ggml_backend_device_i ggml_backend_cpu_device_i = {
+        //    /* .get_buffer_type      = */ ggml_backend_cpu_device_get_buffer_type,
```

Comment:
It's easier to add new functions in this way since it doesn't require updating every backend. The basic ggml-backend interface should remain somewhat stable.